### PR TITLE
Rm err get string table

### PR DESCRIPTION
**Pull request by richsalz**

### CODE REVIEW

**Comment by richsalz** [comment]
See #1379

File: crypto/err/err.c [hunk 1]
```diff
@@ -155,33 +153,13 @@ static int err_string_data_cmp(const ERR_STRING_DATA *a,
...
 static ERR_STRING_DATA *int_err_get_item(const ERR_STRING_DATA *d)
 {
     ERR_STRING_DATA *p = NULL;
-    LHASH_OF(ERR_STRING_DATA) *hash;
 
     CRYPTO_THREAD_read_lock(err_string_lock);
-    hash = get_hash(0, 0);
-    if (hash)
-        p = lh_ERR_STRING_DATA_retrieve(hash, d);
+    if (int_error_hash)
```
Comment (kaduk) [inline crypto/err/err.c]: != NULL?

File: crypto/err/err.c [hunk 2]
```diff
@@ -300,15 +278,15 @@ int ERR_load_ERR_strings(void)
...
 {
-    LHASH_OF(ERR_STRING_DATA) *hash;
-
     CRYPTO_THREAD_write_lock(err_string_lock);
-    hash = get_hash(1, 0);
-    if (hash) {
+    if (int_error_hash == NULL)
+        int_error_hash = lh_ERR_STRING_DATA_new(err_string_data_hash,
+                                                err_string_data_cmp);
+    if (int_error_hash) {
```
Comment (kaduk) [inline crypto/err/err.c]: != NULL

**Comment by kaduk** [comment]
I'm happy to see this in preference to 1379, modulo style nits.

**Comment by richsalz** [comment]
yeah, fixed those with an updated commit.  thanks.

File: util/libcrypto.num [hunk 3]
```diff
@@ -1800,7 +1800,6 @@ EVP_PKEY_meth_get_verify_recover        1785	1_1_0	EXIST::FUNCTION:
 NAME_CONSTRAINTS_check                  1786	1_1_0	EXIST::FUNCTION:
 X509_CERT_AUX_it                        1787	1_1_0	EXIST:!EXPORT_VAR_AS_FUNCTION:VARIABLE:
 X509_CERT_AUX_it                        1787	1_1_0	EXIST:EXPORT_VAR_AS_FUNCTION:FUNCTION:
-ERR_get_string_table                    1788	1_1_0	EXIST::FUNCTION:
```
Comment (mattcaswell) [inline util/libcrypto.num]: This introduces a "gap" in libcrypto.num. Do we need to recreate this before final release? Not a problem for this MR though I think.

**Comment by mattcaswell** [comment]
+1 for merge after beta

**Comment by richsalz** [comment]
merged, thanks.

**Pull request closed without merging**